kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -79,6 +82,12 @@
 
   private static final Logger logger = LoggerFactory.getLogger(AbstractKafkaInputOperator.class);
 
+  static {
+    // We create new consumers periodically to pull metadata (Kafka consumer keeps metadata in cache)
+    // Skip log4j log for ConsumerConfig class to avoid too much noise in application
+    LogManager.getLogger(ConsumerConfig.class).setLevel(Level.OFF);
+  }
+
```

Comment:
Please check if this can be done through slf4j to not introduce the direct log4j dependency in our code.